SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security
+
+The Vega team and community take security bugs in [Vega languages and tools](https://github.com/vega) seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+To report a security issue privately, please use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/vega/.github/security/advisories/new) tab.
+
+A Vega [maintainer](https://github.com/vega/.github/blob/main/project-docs/MAINTAINERS.md) will send a response indicating next steps in handling your report. After the initial reply, the team will keep you informed of the progress towards a fix and announcement, and may ask for additional information or guidance.
+
+You can also report a vulnerability in Vega Javascript packages through the [npm contact form](https://www.npmjs.com/support) by selecting "I'm reporting a security vulnerability", or in Python packages through the [PyPI form](https://pypi.org/security/)
```

Comment:
Let's remove this

Reply:
Removing the NPM and PyPI parts makes sense to avoid splitting the reports up. Did you also mean to remove the part indicating who will follow up on the reports?